src/index.ts

```diff
@@ -11,11 +11,17 @@ const run = async (): Promise<void> => {
 
     const url = getInput('coverageEndpoint');
 
+    if (!url) {
+      warning(
+        'Failed to retrieve `coverageEndpoint` from action. See configuration for instructions on how to add covergeEndpoint to action.'
+      );
+    }
+
     if (url) {
```

Comment:
Is this if statement necessary? since you already check on line 14

Reply:
line 14 is `if(!url){ ...}` and that's to print a warning. the warning won't stop the script from running, just let a user know that url is not defined.

Reply:
ah ok